src/api/services/IncidentsHistory.ts

```diff
@@ -0,0 +1,36 @@
+import { IncidentCommander } from "../axios";
+import { Hypothesis } from "./hypothesis";
+import { Incident } from "./incident";
+
+// expand this list to add more items
+export type IncidentHistoryType =
+  | "evidence.created"
+  | "incident.created"
+  | "responder.created"
+  | "incident_status.updated";
+
+export interface IncidentHistory {
+  id: string;
+  incident_id: string;
+  created_by: CreatedBy;
+  type: IncidentHistoryType;
+  description: string;
+  hypothesis_id?: string;
+  created_at: string;
+  updated_at: string;
+  hypotheses?: Hypothesis[];
+  incidents: Incident[];
+}
+
+export interface CreatedBy {
+  id: string;
+  name: string;
+  avatar: null;
+}
+
+export const getIncidentHistory = async (incidentID: string) => {
+  const res = await IncidentCommander.get<IncidentHistory[]>(
+    `/incident_histories?incident_id=eq.${incidentID}&select=*,created_by(id,name,avatar),hypotheses(*, evidences(*, components(*, config_component_relationships(*)))),incidents(*,created_by(id,name,avatar))&order=created_at.desc`
```

Comment:
We don't need to get this much detail - Just date, createdBy, event, description, component_id, config_id

Reply:
Cool, let me update this, it slipt through as I connected the data and waited for feedback.